test/compilable/test930.d

```diff
@@ -0,0 +1,32 @@
+// https://issues.dlang.org/show_bug.cgi?id=930
+
+/*
+TEST_OUTPUT:
+---
+---
+*/
+
+import core.stdc.stdio;
```

Comment:
You can declare a dummy function with the same signature as printf.

Reply:
I could delete it altogether.

Reply:
and get ridof the import in case it was not clear

Reply:
> I could delete it altogether, I just wanted to preserve the initial bug report as is

yes sorry crossed post. Remove the import and use local dummy please and it's good.

Reply:
I deleted the printf; it's a compilable test anyway.